grant/datastore.go

```diff
@@ -91,6 +93,25 @@ func NewPostgres(databaseURL string, performMigration bool) (*Postgres, error) {
 		return nil, err
 	}
 
+	// setup instrumentation using sqlstats
+	// Create a new collector, the name will be used as a label on the metrics
+	collector := sqlstats.NewStatsCollector("db", db)
+
+	// Register it with Prometheus
+	prometheus.MustRegister(collector)
+
+	if env := os.Getenv("ENV"); env == "production" {
+		// https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraPostgreSQL.Managing.html
+		// we are using db.r5.xlarge	3300 in prod, so setting this to just less (divided by number of instances)
+		// the scaling number is 1000 in terraform
+		db.SetMaxOpenConns(1100)
```

Comment:
could we calculate this via an env var? will have to talk with @hspencer77 about how best to setup... if there is an env already available etc.

Reply:
The latest iteration has this calculation being performed more dynamically.